packages/wallet/frontend/src/lib/api/account.ts

```diff
@@ -115,8 +112,18 @@ const createAccountService = (): AccountService => ({
 
   async list(cookies, include) {
     try {
+      let includeValues = ''
+      if (include?.indexOf('walletAddresses') !== -1) {
+        if (include?.indexOf('walletAddressKeys') !== -1) {
+          includeValues =
+            '?include[]=walletAddresses&include[]=walletAddressKeys'
+        } else {
+          includeValues = '?include=walletAddresses'
+        }
+      }
+
       const response = await httpClient
-        .get(`accounts${include ? `?include=${include}` : ``}`, {
+        .get(`accounts${includeValues}`, {
           headers: {
             ...(cookies ? { Cookie: cookies } : {})
           }
```

Comment:
Since we will take this approach for `searchParams` when listing accounts, can we update `transactions.list` as well?

https://github.com/interledger/testnet/blob/54fe4d6abea39219841cba1c4905ee3bc54b2a0a/packages/wallet/frontend/src/lib/api/transaction.ts#L70-L78

Reply:
There is a story ro change some stuff for transaction list, so I will add this there as well, and do it with that story.
https://github.com/interledger/testnet/issues/543